src/controllers/action-queue/runtime/evaluateAction.ts

```diff
@@ -145,61 +142,27 @@ export async function evaluateAction(context: ExecutionContext, program: ActionP
       }
     }
 
-    case 'wyre-sell': {
-      const { wyreAccountId, nativeAmount, tokenId, walletId } = actionOp
-      const wallet = account.currencyWallets[walletId]
-      const parentCurrencyCode = wallet.currencyInfo.currencyCode
-      const currencyCode = getCurrencyCode(wallet, tokenId)
-
-      const wyreClient = await makeWyreClient({
-        account
-      })
+    // TODO: Remove once we implement action-queue disk data deletion
+    case 'wyre-buy': {
+      const makeExecutionOutput = async (): Promise<ExecutionOutput> => {
+        console.error(new Error(`Using obsolete wyre-buy action`))
```

Comment:
The `new Error` shouldn't be needed. You can print strings to the console.

Reply:
Fixed